rust/tests.yaml

```diff
@@ -1,5 +1,5 @@
 vars:
-  lint_toolchain: 1.75.0
+  lint_toolchain: 1.84.1
```

Comment:
lets update your commit message to say toolchain rather then msrv :)

Reply:
https://github.com/coreos/repo-templates/pull/289/commits/f26101bc0cb8c773884138ec6353d6487e06f670

The commit message is not resolved yet

Reply:
This is still not resolved